### PR TITLE
Fix migration by adding missing field in SubprocessInput migration

### PR DESCRIPTION
**Pull request by Filemon279**

### CODE REVIEW

File: designer/server/src/main/scala/db/migration/V1_046__RenameSubprocessToFragmentDefinition.scala [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    (obj.hcursor downField field).success.flatMap(_.withFocus(update).top).getOrElse(obj)
+  }
+
+  private def updateNodes(array: Json, fun: Json => Json) = fromValues(array.asArray.getOrElse(List()).map(fun))
+
+  private def updateCanonicalNode(node: Json): Json = {
+    node.hcursor.downField("type").focus.flatMap(_.asString).getOrElse("") match {
+      case "SubprocessInput" =>
+        val updateType = updateField(node, "type", _ => Json.fromString("FragmentInput"))
+        updateField(updateType, "outputs", outputs => outputs.mapObject(obj => obj.mapValues(updateCanonicalNodes)))
```
Comment (Filemon279) [inline designer/server/src/main/scala/db/migration/V1_046__RenameSubprocessToFragmentDefinition.scala:39]: This is a field that was missing

File: designer/server/src/main/scala/db/migration/V1_046__RenameSubprocessToFragmentDefinition.scala [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      case "FragmentInput" => updateField(node, "outputs", outputs => outputs.mapObject(obj => obj.mapValues(updateCanonicalNodes)))
```
Comment (Filemon279) [inline designer/server/src/main/scala/db/migration/V1_046__RenameSubprocessToFragmentDefinition.scala:40]: And this is a line that we need to add for this to work after first migration

File: designer/server/src/main/scala/db/migration/V1_043__RenameSubprocessToFragmentDefinition.scala [hunk 3]
```diff
@@ -1,62 +1,11 @@
...
-        case obj if obj.contains(legacyProperty) =>
-          Json.fromJsonObject(obj
-            .add(newProperty, obj(legacyProperty).get)
-            .filterKeys(_ != legacyProperty))
-        case _ => json
-      }.getOrElse(json)
-    }).top.getOrElse(node)
-  }
+  //This migration was missing "outputs" field in SubprocessInput migration so it was replaced with empty migration and fixed in another one.
+  override def updateProcessJson(jsonProcess: Json): Option[Json] = Some(jsonProcess)
```
Comment (piotrp) [inline designer/server/src/main/scala/db/migration/V1_043__RenameSubprocessToFragmentDefinition.scala:9]: You can return None:
```scala
* <li>if we want to continue and fall back to previous json - the implementation should return None</li>
```

**Review by piotrp — APPROVED** [review]

**Comment by piotrp** [comment]
Add changelog entry for 1.11.1

**Comment by Filemon279** [comment]
changelog added, thanks

**Pull request merged**